.gitignore

```diff
@@ -17,3 +17,4 @@ objects/
 
 # project ignores
 packages/web/examples/
+.history
```

Comment:
I don't mind adding more to gitignore but what is .history?

Reply:
it is a folder where vscode plugin saves file copy on every save. This way when doing a lot of changes before commit, deleting piece of code, undo redo ... from time to time saves the day by going back to a prev version of a file:)

https://marketplace.visualstudio.com/items?itemName=xyz.local-history